File: csharp/Svix/Models/ListResponseApplicationOut.cs
// this file is @generated
using System.Text.Json.Serialization;

namespace Svix.Models
{
    public class ListResponseApplicationOut : BaseModel
    {
        [JsonPropertyName("data")]
        public required List<ApplicationOut> Data { get; set; }

        [JsonPropertyName("done")]
        public required bool Done { get; set; }

        [JsonPropertyName("iterator")]
        public required string? Iterator { get; set; }

        [JsonPropertyName("prevIterator")]
        public string? PrevIterator { get; set; }
    }
}

File: javascript/src/models/list_response_application_out.ts
// this file is @generated
import { ApplicationOut, ApplicationOutUtil } from "../models/application_out";

export interface ListResponseApplicationOut {
  data: ApplicationOut[];
  done: boolean;
  iterator: string | null;
  prevIterator?: string | null;
}

export namespace ListResponseApplicationOutUtil {
  export function _fromJsonObject(object: any): ListResponseApplicationOut {
    return {
      data: object["data"].map((item) => ApplicationOutUtil._fromJsonObject(item)),
      done: object["done"],
      iterator: object["iterator"],
      prevIterator: object["prevIterator"],
    };
  }

  export function _toJsonObject(self: ListResponseApplicationOut): any {
    return {
      data: self.data.map((item) => ApplicationOutUtil._toJsonObject(item)),
      done: self.done,
      iterator: self.iterator,
      prevIterator: self.prevIterator,
    };
  }
}
